src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/RenderInputParams/index.tsx

```diff
@@ -6,36 +6,74 @@ import TextField from 'src/components/forms/TextField'
 import { composeValidators, mustBeEthereumAddress, required } from 'src/components/forms/validator'
 import Col from 'src/components/layout/Col'
 import Row from 'src/components/layout/Row'
+import Checkbox from 'src/components/forms/Checkbox'
 
 const RenderInputParams = () => {
   const {
     meta: { valid: validABI },
-  } = useField('abi', { valid: true } as any)
+  } = useField('abi', { valid: true } as Record<string, boolean>)
```

Comment:
This is wrong, if you remove the `as something` it will give you the error. It should have been `value` instead of valid `valid`.

That was my mistake.

 ```suggestion
  } = useField('abi', { value: true })
```

(Thanks TS!!)

Reply:
It's solved, thanks!